docs/tsm-report.cddl

```diff
@@ -1,8 +1,8 @@
-{
-  "auxblob": bstr
-  "manifestblob": bstr
+tsm-report = {
+  ? "auxblob": bstr
+  ? "manifestblob": bstr ; requires service_provider
```

Comment:
should we pair the two then?, e.g.,:

```suggestion
  ? ( "manifestblob": bstr, "service_provider": bstr )
```

Reply:
I considered this but that’s not strictly required since the service_provider may be present without the manifestblob. Maybe I should use a service-provider-group which is (both) / (service provider)

Reply:
The less prose to explain intra-claims-relations, the better.

Reply:
@deeglaze are considering doing more changes?  E.g.:

> Maybe I should use a service-provider-group which is (both) / (service provider)

If not, we can merge this and move forward with the request to IANA.

Reply:
Thanks for the ping. Updated.